docs/containers.conf.5.md

```diff
@@ -971,8 +971,14 @@ On Mac, the default volumes are:
 **provider**=""
 
 Virtualization provider to be used for running a podman-machine VM. Empty value
-is interpreted as the default provider for the current host OS. On Linux/Mac
-default is `QEMU` and on Windows it is `WSL`.
+is interpreted as the default provider for the current host OS.
+
+| Platform | Default                          | Optional |
+| -------- | ---------------------------------| -------- |
+| Linux    | QEMU                             | None     |
+| Windows  | WSL: Windows Subsystem for Linux | HyperV: Windows Server Virtualization |
+| Mac      | vfkit: Apple Hypervisor          | krunkit: Launch machine via libkrun platform, optimized for sharing GPU with the machine  |
```

Comment:
I find this table confusing, as an user it is not obvious what value must be use for `provider` IMO. This either needs to highlight what key to use in the table or I think a list like done in containers.conf file might be better suited, especially if we ever were to add a third provider for whatever reason...

Also it may be helpful to point out that the case of the name does not matter because otherwise a user might be wondering if the case is important given all keys are written with a different case here